service/src/test/java/greencity/mapping/AchievementVOMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package greencity.mapping;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import greencity.ModelUtils;
+import greencity.dto.achievement.AchievementTranslationVO;
+import greencity.dto.achievement.AchievementVO;
+import greencity.dto.achievementcategory.AchievementCategoryVO;
+import greencity.dto.language.LanguageVO;
+import greencity.entity.Achievement;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
```

Comment:
Use **MockitoExtension.class** because you don't need to involve here Spring for testing.

Reply:
fixed